src/main/java/net/cosmocat/marketplace/config/OpenApiConfig.java

```diff
@@ -0,0 +1,41 @@
+package net.cosmocat.marketplace.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import io.swagger.v3.oas.models.servers.Server;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+public class OpenApiConfig {
+
+    @Bean
+    public OpenAPI cosmocatMarketplaceOpenAPI() {
+        Server devServer = new Server();
+        devServer.setUrl("http://localhost:8080");
```

Comment:
That's cool, but you can do it better. Use https://springdoc.org/#swagger-ui-properties - u can do all the configurations on the level of the application.yml